### PR TITLE
[GC-Stress] Added inactive timeout config

### PR DESCRIPTION
**Pull request by agarwal-navin**

### CODE REVIEW

File: packages/test/test-service-load/testConfig.json [hunk 1]
```diff
@@ -5,7 +5,19 @@
             "progressIntervalMs": 5000,
             "numClients": 10,
             "totalSendCount": 1000,
-            "readWriteCycleMs": 10000
+            "readWriteCycleMs": 10000,
+            "optionOverrides":{
+                "tinylicious":{
+                    "configurations":{
+                        "Fluid.GarbageCollection.TestOverride.InactiveTimeoutMs": [5000]
```
Comment (markfields) [inline packages/test/test-service-load/testConfig.json:12]: Are you going to set SessionExpiry override in a separate PR?

**Review by markfields — APPROVED** [review]

File: packages/test/test-service-load/testConfig.json [hunk 2]
```diff
@@ -5,7 +5,19 @@
             "progressIntervalMs": 5000,
             "numClients": 10,
             "totalSendCount": 1000,
-            "readWriteCycleMs": 10000
+            "readWriteCycleMs": 10000,
+            "optionOverrides":{
+                "tinylicious":{
+                    "configurations":{
```
Comment (markfields) [inline packages/test/test-service-load/testConfig.json:11]: So handy!!

File: packages/test/test-service-load/testConfig.json (hunk 1 above)
Comment (agarwal-navin) [reply packages/test/test-service-load/testConfig.json:12]: yes, the idea is to validate current tests with inactive object timeout. Will add session timeout later.

File: packages/test/test-service-load/src/gcDataStore.ts [hunk 3]
```diff
@@ -12,6 +12,11 @@ import { IFluidHandle } from "@fluidframework/core-interfaces";
 import { SharedCounter } from "@fluidframework/counter";
 import { IRunConfig } from "./loadTestDataStore";
 
+/**
+ * How much faster than its parent should a data stores at each level sends ops.
```
Comment (alexvy86) [inline packages/test/test-service-load/src/gcDataStore.ts:16]:
```suggestion
 * How much faster than its parent should a data stores at each level send ops.
```

Comment (agarwal-navin) [reply packages/test/test-service-load/src/gcDataStore.ts:16]: Will fix this in a follow up.

**Pull request merged**